templates/containers/_nav_secondary-v1.html

```diff
@@ -1,6 +1,6 @@
-<ul{% if list_class %} class="{{ list_class }}"{% endif %} class="hover-menu">
-  <li><a{% if level_1 == 'containers' and not level_2 %} class="is-active"{% endif %} href="/containers">Overview</a></li>
-  <li><a{% if level_2 == 'lxd' %} class="is-active"{% endif %} href="/containers/lxd">LXD</a></li>
-  <li><a{% if level_2 == 'kubernetes' %} class="is-active"{% endif %} href="/containers/kubernetes">Kubernetes</a></li>
-  <li><a class="last-item{% if level_2 == 'docker-ubuntu' %} is-active{% endif %}" href="/containers/docker-ubuntu">Docker Engine</a></li>
+<ul class="{% if list_class %}{{ list_class }}{% else %}nav-secondary__menu p-inline-list{% endif %}">
```

Comment:
This is broken - you're missing a classing `"` on line 2 after`class="p-inline-list__link`